org.eclipse.ice.modeling/src/main/java/org/eclipse/ice/modeling/workflowEngine/SuccessCriteria.java

```diff
@@ -0,0 +1,37 @@
+package org.eclipse.ice.modeling.workflowEngine;
+
+public class SuccessCriteria {
+
+	private String successMsg;
```

Comment:
How is this class different from State? Are they doing the same thing, and if not what is their difference?

Reply:
Not sure yet.  Once again, there are a number of things running around in my head that have not crystalized yet.  Initially, in the context of AR, this was to hold a set of "Success view" messages that Mat showed us in one of the message tables he presented.  So I am not yet solid if it is that or some larger context.  It may be possible to handle this with Step State.  Another confusion for me is if success and completion are the same thing.  I am not certain they are.